plugins/woocommerce/tests/php/src/Internal/Logging/RemoteLoggerTest.php

```diff
@@ -90,21 +90,34 @@ public function test_remote_logging_not_allowed( $condition, $setup_callback ) {
 	 */
 	public function remote_logging_disallowed_provider() {
 		return array(
-			'feature flag disabled'   => array(
+			'feature flag disabled' => array(
 				'condition' => 'feature flag disabled',
 				'setup'     => fn() => update_option( 'woocommerce_feature_remote_logging_enabled', 'no' ),
 			),
-			'tracking opted out'      => array(
+			'tracking opted out'    => array(
 				'condition' => 'tracking opted out',
 				'setup'     => fn() => add_filter( 'option_woocommerce_allow_tracking', fn() => 'no' ),
 			),
-			'outdated version'        => array(
-				'condition' => 'outdated version',
-				'setup'     => fn() => WC()->version = '9.0.0',
-			),
-			'high variant assignment' => array(
-				'condition' => 'high variant assignment',
-				'setup'     => fn() => add_filter( 'option_woocommerce_remote_variant_assignment', fn() => 15 ),
+			'outdated version'      => array(
+				'condition'               => 'outdated version',
+				'setup'                   => function () {
+					$version = WC()->version;
+					$next_version = implode(
```

Comment:
```suggestion
					// Generate next major version (e.g., 7.5.1 -> 8.0.0)
					$next_version = implode(
```

`$next_version` calculation looks cryptic, I think adding a comment on what it is or provide some example values may help

Reply:
Oops, didn't realise the auto-merge 😅 

Perhaps a follow-up if it's worth

Reply:
Sorry, I didn't mention auto-merge was enabled. 😅  That's a good point, I'll work on a PR to update the remote logging documentation and will include this in that update.